core/domain/prod_validators.py

```diff
@@ -129,63 +126,7 @@
 ERROR_CATEGORY_SUMMARY_CHECK = 'summary check'
 
 
-class ActivityReferencesModelValidator(base_validators.BaseModelValidator):
```

Comment:
Oops, I reviewed the top file considering that to be a new code! :p

Reply:
Haha, happens :P